README.md

```diff
@@ -20,14 +20,15 @@ Make sure `kubectl` is pointing to your cluster and run `cilium-sysdump` with:
 $ python cilium-sysdump.zip
 ```
 
-Note that by default `cilium-sysdump` will collect logs for the past 30 minutes
-and for all the nodes in the cluster.
+Note that by default `cilium-sysdump` will collect logs all the logs and for
```

Comment:
That sentence reads a bit strange to me, how about "Note that by default `cilium-sysdump` will collect all the logs for all the nodes in the cluster", i.e.:

```suggestion
Note that by default `cilium-sysdump` will collect all the logs for
```

Reply:
Ah, thanks. Missed that.